trento/xml/article_sap_trento.xml

```diff
@@ -1770,11 +1769,11 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
         </mediaobject>
       </figure>  
    <para>
-   With the &suma; settings configured, the &sap; admin can test the connection
-   by clicking <guimenu>Test Connection</guimenu>.
-   If the connection is successful, the Host Details view displays a summary
-   of the available patches and the upgradable packages for each host
-   managed by &suma;:</para>
+   Onc the &suma; settings are configured, the &sap; administrator can test the connection
+   by clicking the corresponding button.
```

Comment:
What do you mean by "the corresponding button"? Is the button labeled "Test"? In that case, use:

```suggestion
   by clicking the <guimenu>Test</guimenu> button.
```

If you don't like that repeats "test" twice, you can use replace the verb "test" with "check".